File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/EnemyInfernalDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyInfernalDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyInfernalDragonKillAnimation() {
        new AnimationTester().testAnimation(new EnemyInfernalDragonKillAnimation());
    }
}

File: test/hud/animations/EnemyInfernalDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyInfernalDragonKillAnimation} from '../../../src/hud/animations/EnemyInfernalDragonKillAnimation.js';

/**
 * Integration test for enemy infernal dragon kill animation on actual hardware.
 */
describe('EnemyInfernalDragonKillAnimation Integration', () => {
    test('playEnemyInfernalDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyInfernalDragonKillAnimation());
    }, 30000);
});
